app/scripts/templates/sign_in.mustache

```diff
@@ -45,8 +45,8 @@
            </form>
 
            <div class="links">
-             <a href="/reset_password" class="left reset-password" data-flow-event="forgot-password">{{#t}}Forgot password?{{/t}}</a>
-             <a href="/#" class="right use-different" data-flow-event="use-different-account">{{#t}}Use a different account{{/t}}</a><br/>
+             <a href="/reset_password{{queryString}}" class="left reset-password" data-flow-event="forgot-password">{{#t}}Forgot password?{{/t}}</a>
+             <a href="/{{queryString}}" class="right use-different" data-flow-event="use-different-account">{{#t}}Use a different account{{/t}}</a><br/>
```

Comment:
the ` use-different` is a messed up case, should we convert this to a button instead of a link, so it cannot be opened in a new tab, or we add `/{{queryString}}&useDifferent=1` ?